src/_P092_DLbus.ino

```diff
@@ -366,7 +372,7 @@ boolean Plugin_092(uint8_t function, struct EventStruct *event, String& string)
           if (nullptr != P092_data) {
             addLog(LOG_LEVEL_INFO, F("Init P092_data_struct ..."));
 
-            if (!P092_data->init(CONFIG_PIN1)) {
+            if (!P092_data->init(CONFIG_PIN1, PCONFIG(0))) {
```

Comment:
I'm not quite sure whether re-initialization (and therefore new GPIO configuration) is done when GPIO is changed in webform. P092_init is set to false in PLUGIN_WEBFORM_SAVE but i think the initialization function is never called because P092_data is != nullptr in this case.

Reply:
I have reviewed the use of P092_init (see last commit)